packages/address-fields-nl/components/PostcodeNLAddressFields.tsx

```diff
@@ -47,6 +47,44 @@ export function PostcodeNLAddressFields(props: AddressFieldsProps) {
 
   return (
     <>
+      {countryFirst && (
+        <FormRow>
+          <SelectElement
+            control={control}
+            name='countryCode'
+            SelectProps={{ autoWidth: true }}
+            variant='outlined'
+            label={<Trans id='Country' />}
+            required={required.countryCode}
+            InputProps={{
+              readOnly,
+              endAdornment: <InputCheckmark show={valid.countryCode} select />,
+            }}
+            options={filterNonNullableKeys(countryList, [
+              'two_letter_abbreviation',
+              'full_name_locale',
+            ]).map(({ two_letter_abbreviation: id, full_name_locale: label }) => ({ id, label }))}
+          />
+
+          {regionList.length > 0 && (
+            <SelectElement
+              control={control}
+              name='regionId'
+              // SelectProps={{ native: true, displayEmpty: true }}
+              variant='outlined'
+              label={<Trans id='Region' />}
+              required
+              InputProps={{
+                readOnly,
+                endAdornment: <InputCheckmark show={valid.regionId} select />,
+              }}
+              options={filterNonNullableKeys(regionList, ['id', 'name']).map(
+                ({ id, name: label }) => ({ id, label }),
+              )}
+            />
+          )}
+        </FormRow>
+      )}
```

Comment:
This whole block is the same as below, so you can just assign it to a variable:


```tsx
const countryFields = (
  <FormRow>
    ...
  </FormRow>
)
```

And conditionally render it;

```suggestion
      {countryFirst && countryFields}
```